sdk/keyvault/azure-security-keyvault-certificates/test/ut/certificate_client_base_test.hpp

```diff
@@ -176,7 +176,9 @@ namespace Azure {
       options.Policy.LifetimeActions.emplace_back(action);
 
       auto response = client.StartCreateCertificate(name, options);
-      auto result = response.PollUntilDone(defaultWait);
+      auto pollResult = response.PollUntilDone(defaultWait);
```

Comment:
Add some validations for the `pollResult` like checking that it has no error completition